deployments/upgrade/rollingUpdate.sh

```diff
@@ -0,0 +1,183 @@
+    local check_standby=$(kubectl -n $NAMESPACE get configmaps $configmaps_name -o json|grep '^\ *"user\.yaml":'|grep "$add_user_yaml")
+    # echo $check_standby
+    if [[ ! ${check_standby} ]];then
+        kubectl -n $NAMESPACE get configmaps $configmaps_name -o json|sed 's/\(\"user.yaml.*\)"$/\1\\n'"$add_user_yaml"'"/'|kubectl -n $NAMESPACE apply -f -
+    fi
+}
+
+function prepare() {
+    set_rollingupdate_mode
+}
+
+function check_deploy_status() {
+    local watchOption=$1
+    local deploy=$2
+    kubectl -n $NAMESPACE rollout status deployment $deploy --watch=$watchOption | grep -i successful 1>/dev/null
+    return $?
+}
+
+function check_rollout_status() {
+    local deploy
+    local watchOption=false
+    if [[ $1 == true ]];then
+        watchOption=$1
+    fi
+    for deploy in $DEPLOYMENTS; do
+        check_deploy_status $watchOption $deploy 1>/dev/null
+        if [[ $? != 0 ]];then
+            echo "deployment [$deploy] was blocked,[$deploy] pods status:"
+            kubectl -n $NAMESPACE get pod|grep $deploy
+            echo "maybe it's being deployed or something wrong,need to be fixed first"
+            exit 1
+        fi
+    done
+    echo 'check:
+    success: all deployments are successful, there are no deployments blocked'
+}
+
+function match_deploy() {
+    # if TARGET_VERSION in
+    local now_version_topology=($(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'rootcoord$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'datacoord$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'indexcoord$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'querycoord$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'indexnode$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'datanode$'),$(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'querynode$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'proxy$') $(echo $DEPLOYMENTS|sed s'/ /\n/'g|grep 'standalone$'))
+    TOPOLOGY=${now_version_topology[@]}
+}
+
+function do_rollout() {
+    echo 'update:'
+    local deploy
+    match_deploy
+    for deploy in ${TOPOLOGY[@]}; do
+        for i in $(echo $deploy|sed 's/,/\n/g');do # care of None
+            deploy_update $i
+            if [[ $? != 0 ]];then
+                exit 1
+            fi
+        done
+    done
+}
+
+function do_rollback() {
+    # not in use now
+    kubectl -n $NAMESPACE rollout history
+}
+
+function Usage() {
+    echo "
+
+    [Warning]: Based on kubectl
+    -n [namespace]                        default: [default]
+                                          The namespace that Milvus is installed in.
+    -i [instance name]                    [required] milvus_instance: The name of milvus instance.
+    -t [target_version]                   target_version: The milvus target version.
+    -w [image_tag]                        image_tag: The new milvus image tag.
+    -o [operation]                        operation: The operation: migrate/rollback.
```

Comment:
In the current implementation， the operation only supports `update`.

This usage info display is not consistent with README.
![image](https://user-images.githubusercontent.com/12268675/217998293-a897b57c-8ea2-4df8-823d-09f2b5385da8.png)